storybook/cypress/e2e/modal.cy.ts

```diff
@@ -42,20 +47,38 @@ describe("Modal", () => {
       .realPress("Tab")
       .get("dso-modal")
       .find("button.dso-primary")
-      .should("have.focus")
-      .realPress("Tab")
-      .get("@dsoModal")
+      .should("have.focus");
+
+    cy.visit("http://localhost:45000/iframe.html?id=core-modal--confirm&args=initialFocus:a")
```

Comment:
ik zou deze naar een eigen test verplaatsen. Deze test is meer voor de focus trap en niet de initiele focus.